dbld/images/helpers/dependencies.sh

```diff
@@ -113,6 +113,9 @@ function install_pip_packages {
         centos-7)
             python -m pip install --upgrade pip==9.0.3
             ;;
+        ubuntu-focal)
```

Comment:
Sorry, I overlooked this. The execution must be also branched in line https://github.com/syslog-ng/syslog-ng/pull/3091/files#diff-707f0993444e3f632cf3ff8d315c2b91R123 (github do not allow me, to comment on that line)

Currently it is working, because something installs python2, I am currently investigating it.

Reply:
Discussed IRL. pip do some "magic" in the background, which I am not comfortable with.

After the `pip3 install --upgrade pip` command:
`/usr/local/bin/pip3` and `/usr/local/bin/pip` are identical, and both of them uses the `#!/usr/bin/python3` shebang.

Since Ubuntu Focal is in pre-release state, I am good with the current state, we will follow up the changes later. (They are not finished with eliminating python2, so I am expecting some.)